### DISCUSSION TITLE
Discord

### DISCUSSION THREAD

**Discussion by tteck** [tteck/Proxmox#1754] [Q&A]
Do you believe it's necessary for Proxmox Helper Scripts to have a presence on Discord?

**Comment by AlexGeek77** [comment]
Personally I like how it works on GitHub. I find Discord less useful for finding other peoples solutions. Then it's easy to link to any release changes here.

**Reply by tteck** [reply]
Thank you for sharing your opinion; I've taken note of it.

**Comment by tteck** [comment]
I've decided, No Discord
